RoosterPlanner.UI/src/app/helpers/filter.pipe.ts

```diff
@@ -18,3 +19,21 @@ export class FilterPipe implements PipeTransform {
     });
   }
 }
+
+@Pipe({name: 'taskFilter'})
+export class TaskFilterPipe implements PipeTransform {
```

Comment:
Dit is een filter op de front-end? Je zoekt dan alleen binnen de opgehaalde taken... Is dat gewenst..

Reply:
In dit geval wel. naar verwachting zullen het aantal taken niet explosief stijgen. Taken worden dus altijd volledig opgehaald uit de DB.